Delete stale comments

pmowrer opened the pull request.

pmowrer committed:
chore: move `index.js` into `/src`

pmowrer committed:
chore: extract partially applied Github API fns

updaterci commented:
undefinedMerging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-22)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

updaterci commented:
undefinedMerging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-22)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-22)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-22)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-22)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-23)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-23)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-23)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-23)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.2.0"></a>
# [1.2.0](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.2.0) (2017-12-23)


### Features

* refactor ([aae5088](https://github.com/Updater/semantic-release-github-pr/commits/aae5088))

pmowrer committed:
fix: clean up stale changelogs when there's no new release

Previously we cleaned up previous PR changelog comments made by this
plugin in the `publish` plugin hook. However, this didn’t work
perfectly as the `plugin` hook only runs if there’s a new release to be
made, leaving the following edge case open:

1. Create a PR with commits that would trigger a release if merged.
2. Observe `semantic-release-github-pr` posting a PR comment with new
release changelog.
3. Update the PR branch such that the commits would no longer trigger a
release if merged.

The PR comment from 2 remains and is now showing incorrect information.

The PR comment is removed following 3.

[ch2522]

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-23)


### Bug Fixes

* clean up stale changelogs when there's no new release ([c2004db](https://github.com/Updater/semantic-release-github-pr/commits/c2004db))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-23)


### Bug Fixes

* clean up stale changelogs when there's no new release ([c2004db](https://github.com/Updater/semantic-release-github-pr/commits/c2004db))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-23)


### Bug Fixes

* clean up stale changelogs when there's no new release ([c2004db](https://github.com/Updater/semantic-release-github-pr/commits/c2004db))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-23)


### Bug Fixes

* clean up stale changelogs when there's no new release ([88dd01e](https://github.com/Updater/semantic-release-github-pr/commits/88dd01e))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-23)


### Bug Fixes

* clean up stale changelogs when there's no new release ([88dd01e](https://github.com/Updater/semantic-release-github-pr/commits/88dd01e))

pmowrer committed:
chore(semnatic-release): bump to `^11.1.0`

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-23)


### Bug Fixes

* clean up stale changelogs when there's no new release ([88dd01e](https://github.com/Updater/semantic-release-github-pr/commits/88dd01e))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-23)


### Bug Fixes

* clean up stale changelogs when there's no new release ([88dd01e](https://github.com/Updater/semantic-release-github-pr/commits/88dd01e))

pmowrer committed:
revert: fix: dogfood on both push and pr

This reverts commit 9173e3c4aff3a4fc7eb747cfa0f5096d516868e6.

`$TRAVIS_PULL_REQUEST` should indicate whether the build was a
"pull request" build. When a branch is pushed to, and it also has
an open PR, Travis will trigger two builds at the same time. If we
run this plugin on both builds, there's a race condition that may
cause a duplicate changelog comment to be posted.

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-24)


### Bug Fixes

* clean up stale changelogs when there's no new release ([c928b54](https://github.com/Updater/semantic-release-github-pr/commits/c928b54))

pmowrer committed:
docs: update README with use of CI env variables

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-24)


### Bug Fixes

* clean up stale changelogs when there's no new release ([c928b54](https://github.com/Updater/semantic-release-github-pr/commits/c928b54))

updaterci commented:
[//]: # (semantic-release-github-pr-semantic-release-github-pr)
Merging this PR will publish the following release:

<a name="1.1.4"></a>
## [1.1.4](https://github.com/Updater/semantic-release-github-pr/compare/v1.1.3...v1.1.4) (2017-12-24)


### Bug Fixes

* clean up stale changelogs when there's no new release ([c928b54](https://github.com/Updater/semantic-release-github-pr/commits/c928b54))
